UnityProject/Assets/Scripts/Objects/StatusDisplay.cs

```diff
@@ -228,20 +241,28 @@ public void ServerPerformInteraction(HandApply interaction)
 				{
 					if (channel == StatusDisplayChannel.DoorTimer)
 					{
-						currentTimerSeconds += 60;
-						if (currentTimerSeconds > 600)
+						if (!AccessRestrictions || AccessRestrictions.CheckAccess(interaction.Performer))
 						{
-							currentTimerSeconds = 1;
-						}
-
-						if (!countingDown)
-						{
-							StartCoroutine(TickTimer());
+							currentTimerSeconds += 60;
+							if (currentTimerSeconds > 600)
+							{
+								currentTimerSeconds = 1;
+							}
+
+							if (!countingDown)
```

Comment:
Explicitly compare to false here. I personally dislike it, but it is part of our code style guidelines.